src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -788,6 +792,9 @@ public override void Init(ICellEditorData owningEditor, bool fresh)
             behaviourEditor.Visible = false;
         }
 
+        processList.ShowToggles = false;
+        processList.ShowSpinners = false;
```

Comment:
These should be set with the Godot Editor.

Reply:
These aren't export fields. Make them such?

Reply:
Yeah, make them exports.